src/deploy/osp_deployer/settings/config.py

```diff
@@ -418,6 +418,28 @@ def __init__(self, settings_file):
         else:
             self.enable_unity_manila_backend = False
 
+        # powermax
+        if backend_settings['enable_powermax_backend'].lower() == 'true':
+            self.enable_powermax_backend = True
+            self.powermax_backend_name = backend_settings['powermax_backend_name']
+            self.powermax_san_ip = backend_settings['powermax_san_ip']
+            self.powermax_san_login = backend_settings[
+                'powermax_san_login']
+            self.powermax_san_password = backend_settings[
+                'powermax_san_password']
+            self.powermax_protocol = backend_settings[
+                'powermax_protocol']
```

Comment:
Probably want to check that protocol being set is a valid one. ( especially with mixed case in iSCI)